zeeguu/core/feed_handler/rssfeed.py

```diff
@@ -6,8 +6,9 @@
 
 
 class RSSFeed(FeedHandler):
-    def __init__(self, url: str, feed_type: int):
-        super().__init__(url, feed_type)
+    def __init__(self, url: str, feed_type: int, is_stored_db: bool = False):
+        super().__init__(url, feed_type, is_stored_db)
+        logp(f"Created RSS Source ({self.url})")
```

Comment:
I removed this logline because it was confusing. Given that our feeds are mapped on to the DB, the "created" message should only appear when a new object is added to the DB. Or is this the case here?

Reply:
True, that isn't necessarily a creating a source, but rather using the RSS handler, so I will rephrase it to "Using RSS Handler" and "Using Newspaper Handler" what do you think ?